ranger/core/actions.py

```diff
@@ -489,7 +489,10 @@ def move(self,  # pylint: disable=too-many-locals,too-many-branches,too-many-sta
             if narg is not None:
                 mode = narg
             tfile = self.thisfile
-            selection = self.thistab.get_selection()
+            if not kw.get('selection', True):
```

Comment:
Could you switch the `if` branches to eliminate too much negation? It quickly gets confusing.